sdk/src/Core/Amazon.Runtime/Credentials/AssumeRoleImmutableCredentials.cs

```diff
@@ -41,15 +41,20 @@ public AssumeRoleImmutableCredentials(string awsAccessKeyId, string awsSecretAcc
         {
             if (string.IsNullOrEmpty(token)) throw new ArgumentNullException("token");
             Expiration = expiration;
+            
         }
 
+        public AssumeRoleImmutableCredentials(string awsAccessKeyId, string awsSecretAccessKey, string token, DateTime expiration, string accountId) : this (awsAccessKeyId, awsSecretAccessKey, token, expiration)
```

Comment:
Constructor needs docs. For account be sure to add any formatting requirements like no hyphens.